error/response.go

```diff
@@ -11,7 +11,7 @@ type (
 )
 
 const (
-	ApplicationError Type = "ApplicationError"
+	ApplicationError Type = "Valid"
```

Comment:
do you want to rename ApplicationError to ValidError?

Reply:
whoops, will revert